Cannot update hotspot from frontend

Hi!

First of all, love the project! Fantastic work!

I'm running into an issue with updating the hotspot settings from the frontend. The call returns a 400 Bad Request. The backend logs: `Bad body: json: unknown field "type"`.

This is due to the struct in the backend being:
```go
type hotspotUpdateOptions struct {
	Password *string `json:"password"`
	Ssid     string  `json:"ssid"`
}
```
The request that is send from the frontend is:
```json
{"type":"hotspot","password":"password","ssid":"ssid"}
```
I'd be happy to submit the fix myself, but I'm not sure if this needs to be fixed in the frontend by removing the type, or adding the type in the backend.

Hi!

Attempting a fix but running into an issue. I've never done cross compiling before so I might just be overlooking something. When running `make crossdeb` I get:
```
make[2]: Entering directory '/home/me/Documents/dev/fruitnanny-api'
dh_installsystemd --name=fruitnanny-api
dh_installsystemd --name=fruitnanny-update --no-enable --no-start
dh_installsystemd --name=fruitnanny-upgrade --no-enable --no-start --no-stop-on-upgrade
make[2]: Leaving directory '/home/me/Documents/dev/fruitnanny-api'
   dh_perl
   dh_link
   dh_strip_nondeterminism
   dh_compress
   dh_fixperms
   dh_missing
   dh_strip
Can't exec "arm-linux-gnueabihf-strip": No such file or directory at /usr/share/perl5/Debian/Debhelper/Dh_Lib.pm line 483.
dh_strip: arm-linux-gnueabihf-strip --remove-section=.comment --remove-section=.note debian/fruitnanny-api/usr/bin/fruitnanny failed to execute: No child processes
dh_strip: arm-linux-gnueabihf-strip --remove-section=.comment --remove-section=.note debian/fruitnanny-api/usr/bin/fruitnanny returned exit code 2
dh_strip: Aborting due to earlier error
make[1]: *** [debian/rules:16: binary] Error 255
make[1]: Leaving directory '/home/me/Documents/dev/fruitnanny-api'
dpkg-buildpackage: error: fakeroot debian/rules binary subprocess returned exit status 2
make: *** [Makefile:60: crossdeb] Error 2
```
Any ideas? Thanks!

--- a/internal/api/api.go
+++ b/internal/api/api.go
@@ -448,6 +448,7 @@ func (s *server) readHotspot() http.HandlerFunc {
 }
 
 type hotspotUpdateOptions struct {
+	Type     string	 `json:"type"`
 	Password *string `json:"password"`
 	Ssid     string  `json:"ssid"`
 }

